modin/pandas/dataframe.py

```diff
@@ -2498,6 +2498,9 @@ def __setattr__(self, key, value):
             pass
         elif key in self and key not in dir(self):
             self.__setitem__(key, value)
+            # Note: Return so that we don't keep track of another attribute. Instead
+            # column names and such will be returned when __getattr__ is called
+            return
```

Comment:
```suggestion
            # Use __setitem__  to update an existing column with a new value.
            # In this case, we don't need to set the column label's attribute
            # explicitly, because __getattr__ will get the updated column
            # whenever it's needed. Return early to avoid setting the column
            # label's attribute later in this method.
            self.__setitem__(key, value)
            return
```
I think this is a clearer, if somewhat wordy, explanation.

Reply:
@devin-petersohn had a suggestion from earlier that seemed to be fairly straightforward. Do you guys have a preference?

Reply:
I vote for his suggestion :)